File: content/create-symlink/abs.go
package main
import (
   "os"
   "path/filepath"
)
func main() {
   var s_pub = os.Getenv("PUBLIC")
   var s_old, s_new string
   // round 1
   s_old, _ = filepath.Abs("abs.go")
   s_new = filepath.Join(s_pub, "Music", "abs.go")
   os.Symlink(s_old, s_new)
   // round 2
   s_old, _ = filepath.Abs("chdir.go")
   s_new = filepath.Join(s_pub, "Videos", "chdir.go")
   os.Symlink(s_old, s_new)
}

File: content/create-symlink/abs.php
<?php
$s_pub = getenv('PUBLIC');
$s_sep = DIRECTORY_SEPARATOR;
# round 1
$s_old = realpath('abs.php');
symlink($s_old, $s_pub . $s_sep . 'Music' . $s_sep . 'abs.php');
# round 2
$s_old = realpath('chdir.php');
symlink($s_old, $s_pub . $s_sep . 'Videos' . $s_sep . 'chdir.php');
